examples/platform/silabs/BaseApplication.cpp

```diff
@@ -261,7 +261,7 @@ CHIP_ERROR BaseApplication::Init()
      * Wait for the WiFi to be initialized
      */
     ChipLogProgress(AppServer, "APP: Wait WiFi Init");
-    while (!wfx_hw_ready())
+    while (!IsHardwareReady())
```

Comment:
without the class name, it's not evident what hardware we are waiting on. In mind, it also only makes sense that we are waiting on hardware setup for ncp/rcp cases. Do you know what we are waiting on for 917 soc case?

Reply:
I think the name is deceiving - we aren't really waiting for the hardware. We are waiting for the Wifi init to have completed. I'll renamed the function.

Reply:
renamed function for clarity